src/modules/roc_audio/mixer.cpp

```diff
@@ -28,9 +28,18 @@ sample_t clamp(const sample_t x) {
 
 } // namespace
 
-Mixer::Mixer(core::BufferPool<sample_t>& pool, size_t frame_size)
+Mixer::Mixer(core::BufferPool<sample_t>& pool, 
+             core::nanoseconds_t frame_length, 
+             size_t sample_rate, 
+             packet::channel_mask_t channel_mask)
     : valid_(false) {
-    roc_log(LogDebug, "mixer: initializing: frame_size=%lu", (unsigned long)frame_size);
+    size_t frame_size = packet::ns_to_size(frame_length, sample_rate, channel_mask);
+    roc_log(LogDebug, "mixer: initializing: frame_size=%lu", frame_size);
```

Comment:
`(unsigned long)` is required in roc_log() due of `%lu`.

Reply:
done.